spec/claCheckSpec.js

```diff
@@ -31,9 +31,15 @@ const sheetsCredentials =
 
 describe('CLA check github action Module for Oppia', () => {
   beforeEach(async () => {
-    process.env.SHEETS_CRED = sheetsCredentials;
-    process.env.SHEETS_TOKEN = sheetsToken;
-    spyOn(core, 'getInput').and.returnValue('sample-token');
+    spyOn(core, 'getInput').and.callFake((val) =>{
+      if (val === 'repo-token') {
+        return 'sample-token';
+      } else if (val === 'sheets-cred') {
+        return sheetsCredentials;
+      } else if (val === 'sheets-token') {
+        return sheetsToken;
+      }
```

Comment:
Should this function be separated so that we get rid of the duplicity?

Reply:
No, Since the function is required to return different values for one of the tests.